lib/identity_doc_auth/lexis_nexis/responses/true_id_response.rb

```diff
@@ -27,15 +27,15 @@ def successful_result?
         def error_messages
           return {} if successful_result?
 
-          if true_id_product.present?
+          if true_id_product.present? && true_id_product[:AUTHENTICATION_RESULT].present?
```

Comment:
I know that `.present?` wasn't added for the first time here, but for good practice can we add the activesupport require for this at the top of the file?

```
require 'active_support/core_ext/object/blank'
```

Reply:
Also another idea is to chain this condition?

```suggestion
          if true_id_product&.dig(:AUTHENTICATION_RESULT).present?
```
(we could do `&.[](:AUTHENTICATION_RESULT)` but I think that looks weird

Reply:
Yeah, I can update.